.gitignore

```diff
@@ -33,3 +33,26 @@ tmp/*
 scripts/yarn.lock
 /.eslintcache
 /.stylelintcache
+
+website/docs/*
```

Comment:
@juzhiyuan This change means we are ignoring all files that should come under "General" docs.